pkg/cluster/cluster_app.go

```diff
@@ -14,9 +14,9 @@ func CreateClusterApp(
 	appIf argoapp.ApplicationServiceClient,
 	argocdNs string,
 	clusterName string,
-	repoUrl string,        // source repo
-	repoRevision string,   // source revision
-	repoPath string,       // source path
+	repoUrl string, // source repo
```

Comment:
Looks like we are using two different code formatters / different versions of `go fmt` / IDE code formatting policies - causing some trivial/ spurious changes to be highlighted.